ibis/expr/datatypes/value.py

```diff
@@ -312,8 +316,47 @@ def normalize(typ, value):
             elif typ.is_multipolygon():
                 return tuple(normalize(dt.polygon, item) for item in value)
         return _WellKnownText(value.wkt)
+    elif (is_timestamp := typ.is_timestamp()) or typ.is_date():
+        import pandas as pd
```

Comment:
We may want to use lazy dispatch here in the future.

Reply:
Makes sense!